docs/defaults/show/decade.md

```diff
@@ -52,10 +52,10 @@ The below is an example config.yml extract with some Template Variables added in
 libraries:
   TV Shows:
     metadata_path:
-      - pmm: decade
+      - pmm: show/decade
```

Comment:
decade is correct.

show/decade is not needed.

```
just FYI all you need is - pmm: decade 
same for - pmm: golden and - pmm: oscars etc
```
Per Yozora
https://discord.com/channels/822460010649878528/1042087472424829038/1042092609226674176